src/main/host/thread.rs

```diff
@@ -638,12 +631,12 @@ mod export {
     }
 
     #[no_mangle]
-    pub unsafe extern "C" fn thread_getProcess(thread: *const Thread) -> *const ProcessRefCell {
+    pub unsafe extern "C" fn thread_getProcess(thread: *const Thread) -> *const Process {
         let thread = unsafe { thread.as_ref().unwrap() };
         Worker::with_active_host(|host| {
             let process = host.process_borrow(thread.process_id).unwrap();
-            let process: &ProcessRefCell = &process;
-            process as *const _
+            let p = &*process.borrow(host.root()) as *const _;
+            p
```

Comment:
Maybe cast to the type explicitly here just to be safe (and then I think you can skip the pointer cast)?

```rust
&*process.borrow(host.root()) as &Process
```

Reply:
Added an explicit type; we still need the pointer cast though